Tweaks required to compile project

lod opened the pull request:
Fixes #2 - Missing build dependencies
Fixes cstdint error with 64bit ints by upgrading compiler release standard.
Fixes cmake multiple library error by removing libusb0 support
Fixes sprintf missing library in src/lib/seekthermal/models/aaa/usb/configreceive.cpp

lod committed:
Tweaks required to compile project

audunmg commented:
I also had to make these changes, and also I had to #include <cmath> in src/bin/gui/view.cpp

Works nicely on Debian with libboost-chrono-dev, no idea how it works on Ubuntu though.

vitcozzolino committed:
Move EXTRA_CXX_FLAGS outside remake_project()

Apparently ReMake ignores the EXTRA_CXX_FLAGS defined inside remake_project(...). To temporary fix the issues, I suggest declaring the flags following the standard Cmake approach.

lod committed:
Merge pull request #1 from vitcozzolino/patch-1

Move EXTRA_CXX_FLAGS outside remake_project()
